elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -264,9 +264,20 @@ elytron.security-domain.realms.principal-transformer=A principal transformer to
 elytron.security-domain.realms.role-decoder=A RoleDecoder reference to be associated with the realm.
 elytron.security-domain.realms.role-mapper=A RoleMapper reference to be associated with the realm.
 elytron.security-domain.trusted-security-domains=The list of security domains that are trusted by this security domain.
+elytron.security-domain.trusted-virtual-security-domains=The list of virtual security domains that are trusted by this security domain.
 elytron.security-domain.outflow-security-domains=The list of security domains that the security identity from this domain should automatically outflow to.
 elytron.security-domain.outflow-anonymous=When outflowing to a security domain if outflow is not possible should the anonymous identity be used?  Outflowing anonymous has the effect of clearing any identity already established for that domain.
 
+###########################
+# Virtual Security Domain #
+###########################
+
+elytron.virtual-security-domain=A virtual security domain definition.
+elytron.virtual-security-domain.add=The add operation for a virtual security domain definition.
+elytron.virtual-security-domain.remove=The remove operation for a virtual security domain definition.
+elytron.virtual-security-domain.outflow-security-domains=The list of security domains that the security identity from this virtual domain should automatically outflow to.
+elytron.virtual-security-domain.outflow-anonymous=When outflowing to a security domain if outflow is not possible should the anonymous identity be used?  Outflowing anonymous has the effect of clearing any identity already established for that domain.
```

Comment:
> outflow is not possible

Which cases are covered by this? I see two possible and a clarification would help
- There is an outflow security domain specified which is not configured to trust this virtual security domain
- The outflowed identity is not known in the trusting security domain

Reply:
Clarified this, thanks!